shared/profile/wot-author/index.tsx

```diff
@@ -302,6 +319,180 @@ export const Question2 = (props: Question2Props) => {
+      >
+        <Kb.Box2 direction="horizontal" alignItems="center" alignSelf="stretch" style={styles.outsideBox}>
+          {props.firstDraft ? (
+            <Kb.Text type="BodyBig" style={styles.reviewInstruction}>
+              Review what{' '}
+              <Kb.ConnectedUsernames
+                usernames={props.voucherUsername}
+                type="BodyBig"
+                inline={true}
+                colorFollowing={true}
+                colorBroken={true}
+              />{' '}
+              wrote about you:
+            </Kb.Text>
+          ) : (
+            <Kb.Text type="BodyBig" style={styles.reviewInstruction}>
+              Review{' '}
+              <Kb.ConnectedUsernames
+                usernames={props.voucherUsername}
+                type="BodyBig"
+                inline={true}
+                colorFollowing={true}
+                colorBroken={true}
+              />
+              's edits:
+            </Kb.Text>
+          )}
+        </Kb.Box2>
+        <Kb.Box2 direction="horizontal" alignSelf="stretch" alignItems="center">
+          <Kb.Box2
+            key="colorbar"
+            direction="vertical"
+            alignSelf="stretch"
+            style={{
+              backgroundColor: verificationTypeColors[props.verificationType] ?? Styles.globalColors.white,
+              flexShrink: 0,
+              marginRight: Styles.globalMargins.small,
+              width: 4,
+            }}
+          />
+          <Kb.Box2 direction="vertical" alignSelf="stretch" style={Styles.globalStyles.flexOne}>
+            <Kb.Box2 direction="horizontal" alignSelf="stretch" alignItems="center">
+              <Kb.Text type="Body" style={Styles.globalStyles.flexOne}>
+                {props.verificationType === 'other'
+                  ? props.otherText
+                  : verificationTypeText(props.verificationType, props.voucheeUsername)}
+              </Kb.Text>
+            </Kb.Box2>
+            {props.verificationType === 'proofs' &&
+              props.proofs.map(proof => (
+                <Kb.Box2
+                  key={`${proof.type}:${proof.value}`}
+                  direction="horizontal"
+                  alignSelf="stretch"
+                  alignItems="center"
+                >
+                  <ProofSingle {...proof} />
+                </Kb.Box2>
+              ))}
+          </Kb.Box2>
+        </Kb.Box2>
+        <Kb.Checkbox
+          checked={checkedVerificationType}
+          onCheck={setCheckedVerificationType}
+          style={styles.reviewCheckbox}
+          label="This is true"
+        />
+        <Kb.Box2 direction="horizontal" alignSelf="stretch" alignItems="center">
+          <Kb.Box2 key="colorbar" direction="vertical" alignSelf="stretch" style={styles.reviewColorbar} />
+          <Kb.Box2 direction="vertical" alignSelf="stretch" style={Styles.globalStyles.flexOne}>
+            <Kb.Box2 direction="horizontal" alignSelf="stretch" alignItems="center">
```

Comment:
I haven't spun up the story yet, but what does `stretch `do here that `fullWidth` doesn't? 

`fullWidth={true}` giving you `{ flexGrow: 1, width: '100%' }`

Reply:
I looked at mobile and desktop and see no difference between using `fullWidth={true}`, `alignSelf="stretch"`, both, nor neither.